js/actions/types.js

```diff
@@ -39,17 +45,20 @@ export type Action =
   | NavigationAction
   | ProfileActions
 
+/* eslint-disable no-use-before-define */
```

Comment:
pls explain why the ignore is needed

Reply:
The problem is described here https://github.com/babel/babel-eslint/issues/485

Reply:
yes pls reference the issue in a comment